README.md

```diff
@@ -1 +1,40 @@
-# Hello-World
\ No newline at end of file
+Hello Microverse Project 
+
+> Model 1 Day # 2 - Simple HTML and CSS Project
+
+
+## Built With
+
+- HTML
+- css
+
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/henrycode460/Hello-World/)
+- 
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you input your project GitHub issues link into the parenthesis 

### EXAMPLE
`[issues page](https://github.com/henrycode460/Hello-World/issues)`